helios-authentication/src/main/java/com/spotify/helios/auth/AuthenticationPlugin.java

```diff
@@ -45,8 +45,6 @@
 
   ServerAuthentication<C> serverAuthentication();
 
-  ClientAuthentication<C> clientAuthentication();
```

Comment:
I think that would make sense; I initially thought it would be simplest for plugin-implementors to implement both at the same time / in the same "package".